### ISSUE TITLE
Calling `getAvailableOnlineStreamDataTypes` throws `PolarBleSdk.PolarErrors.notificationNotEnabled` with the H9

### ISSUE THREAD

**Issue by Rexios80** [polarofficial/polar-ble-sdk#359]
**Platform on which you observed the bug:** 
- [x] Android 
- [x] iOS
- [ ] Other
- [ ] Platform is not relevant for this bug

**Device on which you observed the bug:**
- [ ] Polar OH1
- [ ] Polar Verity Sense
- [ ] Polar H10
- [x] Polar H9
- [ ] Other
- [ ] Device is not relevant for this bug

**Describe the bug**
Getting `PolarBleSdk.PolarErrors.notificationNotEnabled` when calling `getAvailableOnlineStreamDataTypes`

This is confirmed working for the H10 and Verity

**How to Reproduce**
Call `getAvailableOnlineStreamDataTypes` for a Polar H9

**Expected behavior**
It works like with the H10 and Verity

**Screenshots and logs**
<img width="1091" alt="Screenshot 2023-04-07 at 11 39 31 AM" src="https://user-images.githubusercontent.com/7896519/230636843-bc8d2f44-aad8-46b5-96eb-f131633590d2.png">

**Additional Information**
Not sure how useful this is, but here's the code for the plugin: https://github.com/Rexios80/polar
The example app in that repo can reproduce the issue

**Comment by samulimaa** [comment]
Hi, `getAvailableOnlineStreamDataTypes` is part of the Polar Online Streaming API which is only supported by Verity Sense, H10 and OH1 devices. But I think that the thrown error could be improved to reveal the actual cause.

**Comment by Rexios80** [comment]
That makes sense, except heart rate is part of streaming now which is why I expect that call to not fail on any device that supports heart rate streaming. I would expect `getAvailableOnlineStreamDataTypes` to return heart rate on every Polar device. I suppose I can just assume that heart rate is available, but it's still weird.

**Comment by Rexios80** [comment]
Also, `sdkFeatureReady` is returning `onlineStreaming` when connected to the H9. I'm waiting for that to even make the `getAvailableOnlineStreamDataTypes` call

<img width="481" alt="Screenshot 2023-04-19 at 6 29 11 AM" src="https://user-images.githubusercontent.com/7896519/233047758-2173ad06-1934-4b44-a0cb-70a664bbbea6.png">

**Comment by Rexios80** [comment]
@samulimaa Is a fix for this planned?

**Comment by Rexios80** [comment]
@samulimaa This seems like it would be trivial to fix. Should we except a solution for this soon or work around it?

**Comment by Rexios80** [comment]
@JOikarinen @palmqvisti @TeemuLauPolar @andYgnite @jimmyzumthurm @pth-pe-gh 

This was reported 2 years ago and has had no response since. I now have a user of my plugin reporting this issue.

Possible solutions:
- Remove `onlineStreaming` from `sdkFeatureReady`
- Fix `getAvailableOnlineStreamDataTypes`
- Close this issue and I'll accept you don't want to fix it

**Comment by rkangast** [comment]
Hello @Rexios80 

We are sorry that you had to wait for a response for too long. We are now investigating this issue. A probable solution is to move the checking of hr service availability in getAvailableOnlineStreamDataTypes method away from  blePMDClient.clientReady rxjava chain. I cannot promise that this fix will be in our next SDK release, but most likely it will be in the next subsequent release.

**Comment by rkangast** [comment]
Hello @Rexios80 

Some interim information on this issue. I ended up doing a new API method "getAvailableHRServiceDataTypes" to solve this problem. It returns similar set of data as getAvailableOnlineStreamDataTypes method, so it should be straightforward to use instead. The getAvailableOnlineStreamDataTypes method could not be modified without bringing potential problems to other customers. The new API will be released in the next SDK release.

**Comment by Rexios80** [comment]
@rkangast Did you also remove "onlineStreaming" from the list of available data types form the H9?

**Comment by rkangast** [comment]
@Rexios80 

Yes I did, there is only "HR" as H9 does not support our PMD service that the other online measurement types use. But you can use H9 to measure online HR by calling startHrStreaming from PolarOnlineStreaming API. That API is using Bluetooth heart rate service the measure HR (for all our devices supported by SDK).

**Comment by rkangast** [comment]
Hello @Rexios80 

polar-ble-sdk version 6.3.0 has been released and it has the [new API](https://polarofficial.github.io/polar-ble-sdk/polar-sdk-android/com/polar/sdk/api/PolarOnlineStreamingApi.html#getAvailableHRServiceDataTypes(java.lang.String)) in place.
